moveit_ros/visualization/CMakeLists.txt

```diff
@@ -79,6 +79,16 @@ pluginlib_export_plugin_description_file(rviz_common robot_state_rviz_plugin_des
 #  add_rostest(test/moveit_joy.test)
 #endif()
 
+ament_export_libraries(moveit_motion_planning_rviz_plugin_core)
+ament_export_libraries(moveit_motion_planning_rviz_plugin)
+ament_export_libraries(moveit_planning_scene_rviz_plugin_core)
+ament_export_libraries(moveit_planning_scene_rviz_plugin)
+ament_export_libraries(moveit_robot_state_rviz_plugin)
+ament_export_libraries(moveit_robot_state_rviz_plugin_core)
+ament_export_libraries(moveit_rviz_plugin_render_tools)
+ament_export_libraries(moveit_trajectory_rviz_plugin_core)
+ament_export_libraries(moveit_trajectory_rviz_plugin)
+ament_export_include_directories(include)
```

Comment:
Shouldn't this list the actual include directories of the plugins?

Reply:
No, this will export the main installed `include` directory